submissions/edgaewgeg.md

```diff
@@ -0,0 +1,11 @@
+* submitter: @edgaewgeg
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
+* @Asthma is a long term inflammatory disease of the airways.
+* @Asthma is characterized by increased contractility of airway smooth muscles, leading to narrowing of the airway and wheezing.
+* Inflammation in @Asthma occurs as a result of an immune response to environmental triggers.
+* There is no cure for @Asthma, only treatment for symptoms.
```

Comment:
That really sucks.

Reply:
yeah its rough but thanks to science natural selection cant get me down